### DISCUSSION TITLE
Images gone missing after latest update

### DISCUSSION THREAD

**Discussion by Its4Nik** [outline/outline#7682] [Self Hosting] [answered]
Since I updated to the latest version of outline all my images have gone missing, new image uploads still work and seem to be ok,

**To Reproduce**
1. Update to v0.80.2
3. Look for images from older versions
4. They have gone missing

**Expected behavior**
Well i expected to see the images

**Screenshots**
![image](https://github.com/user-attachments/assets/43789c8e-8a93-457d-b5f5-a36a0a4e522a)
![image](https://github.com/user-attachments/assets/a4bc8bba-0fdf-4c8b-a000-9784528eba19)

**Outline (please complete the following information):**
- Install: self hosted
- Version: 840db4692ebc62b1c15b5816333779e9f5f7d4a5

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Zen (Firefox)
 - Version 1.0.1-a.5 (64-bit)

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-671/images-gone-missing-after-latest-update">OLN-671 Images gone missing after latest update</a></p>

**Comment by Its4Nik** [comment, marked as answer]
Okay it seemed like it was a problem with the postgresql, I sticked the version to 16.4 and now images load.

**Reply by mzeecedric** [reply]
I can confirm this issue since I was on latest release as well and switched postgres to v16. Outline-Wiki can not connect to postgres, postgres does not start...

This is in the postgres logs:

```
2024-09-28 15:34:58.960 UTC [1] FATAL:  database files are incompatible with server

2024-09-28 15:34:58.960 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 16, which is not compatible with this version 17.0 (Debian 17.0-1.pgdg120+1).

PostgreSQL Database directory appears to contain a database; Skipping initialization
```

**Comment by Its4Nik** [comment]
AS prove here the 2 images which wouldnt load:

![image](https://github.com/user-attachments/assets/ea506849-6d76-4676-8d5f-201a3ff6f15b)
![image](https://github.com/user-attachments/assets/e29ddee8-ca78-49d0-9e0b-29ec6e067d68)
